gto-support-db/src/main/java/org/ccci/gto/android/common/db/AbstractDao.java

```diff
@@ -236,10 +239,9 @@ public final Cursor getCursor(@NonNull final Class<?> clazz, @NonNull final Stri
                 projection[i] = projection[i].contains(".") ? projection[i] : prefix + projection[i];
             }
 
-            // prefix an un-prefixed orderBy field
-            if (orderBy != null && !orderBy.contains(".")) {
-                orderBy = prefix + orderBy;
-            }
+            // prefix un-prefixed clauses
+            addPrefixToFields(orderBy, prefix);
```

Comment:
and you aren't even capturing the response, let me rework how groupBy is handled, I'd like to use the Field object for those and avoid having to manually prefix strings.